aspnetcore/security/authorization/iauthorizationpolicyprovider.md

```diff
@@ -100,17 +100,17 @@ internal class MinimumAgePolicyProvider : IAuthorizationPolicyProvider
     // to be embedded in the policy names. This is abstracted away from developers
     // by the more strongly-typed attributes derived from AuthorizeAttribute
     // (like [MinimumAgeAuthorize()] in this sample)
-    public Task<AuthorizationPolicy> GetPolicyAsync(string policyName)
```

Comment:
`GetPolicyAsync` doesn't actually do anything async, so it doesn't need the async keyword. `Task.FromResult` is useful for scenarios like this where an async interface is being implemented with a synchronous implementation.

It's not a big difference, but making this method async would trigger diagnostic CS1998 and `Task.FromResult` is the recommended solution (https://stackoverflow.com/a/13254787).